src/styles/navstyles.css

```diff
@@ -16,11 +16,49 @@
   
   html,
   body {
-    font-family: "Segoe UI Variable Display", "Segoe UI", -apple-system, Roboto,
-      sans-serif, serif;
+    font-family: "Roboto Slab", serif;
     margin: 0;
-    /* overflow-x: hidden; */
+    font-weight: normal;
   }
+
+  header {
```

Comment:
Could you add a comment for all the styles you have in here? That way it's easier to understand what parts of the site they change without having to go back and forth between the javascript and css files. Same kind of comment you have on line 51 is what I'm looking for.